packages/shared/lib/core/layer-2/actions/loadEvmAddressForSelectedAccount.ts

```diff
@@ -1,22 +1,15 @@
 import { getSelectedAccount } from '@core/account/stores'
 import { Ledger } from '@core/ledger/classes'
-import { network } from '@core/network'
-import { get } from 'svelte/store'
 
-export async function loadEvmAddressForSelectedAccount(): Promise<void> {
+export async function loadEvmAddressForSelectedAccount(coinType: number): Promise<void> {
     try {
         const account = getSelectedAccount()
-        if (!account) {
+        if (!account || coinType === undefined) {
```

Comment:
we specify the function param as `number` which doesnt include undefined. we can remove this check

Reply:
updated ✅